src/main/webapp/queue/ta.js

```diff
@@ -82,12 +82,12 @@ function getQueue() {
         queueListElement.appendChild(createListElement(studentEmail));
     });
 
+
+    const beinghelpedElem = document.getElementById('beingHelped');
```

Comment:
I think I saw this change in Bianca's CL as well, make sure to coordinate with her